docs/api/Rectangle.md

```diff
@@ -47,6 +47,25 @@ Adjust the rectangle by scaling it. The `scaleHeight` argument can be ommited to
 
 Return this rectangle (useful if you want to chain the calls).
 
+## Change the coordinates basis
+
+```javascript
+var newRect = rect.changeBasis({
+  from: layerA,
+  to: layerB,
+})
+```
+
+Each layer defines its own coordinates basis. You can change that basis from one layer to the other with `changeBasis`.
+
+| Parameters                                            |                                                                   |
+| ----------------------------------------------------- | ----------------------------------------------------------------- |
+| change<span class="arg-type">object - required</span> |                                                                   |
+| change.from<span class="arg-type">Layer</span>        | The layer in which the rectangle's coordinates are expressed.     |
+| change.to<span class="arg-type">Layer</span>          | The layer in which the rectangle's coordinates will be expressed. |
+
+Both `from` and `to` can be ommited (but not at the same time) to change the basis from/to the Page coordinates.
```

Comment:
ah - I was confused until I read this :) I would structure this slightly differently if possible - it would be better if this was explicit in the parameters e.g. `default - currentPage` or something like that. I missed this note for a bit when evaluating the API, so I was left wondering what will happen if I leave them out